packages/ra-ui-materialui/src/layout/Sidebar.js

```diff
@@ -11,7 +11,7 @@ export const CLOSED_DRAWER_WIDTH = 55;
 const useStyles = makeStyles(
     theme => ({
         drawerPaper: {
-            position: 'relative',
+            position: 'sticky',
```

Comment:
this completely breaks the layout on iE11 (which is otherwise pretty well supported)